src/Illuminate/Database/Schema/Builder.php

```diff
@@ -68,6 +68,27 @@ public function hasColumn($table, $column)
 	}
 
 	/**
+	 * Determine if the given table has given columns.
+	 *
+	 * @param  string  $table
+	 * @param  array   $columns
+	 * @return bool
+	 */
+	public function hasColumns($table, array $columns)
+	{
+		$tableColumns = $this->getColumnListing($table);
+
+		array_map('strtolower', $tableColumns);
```

Comment:
yeah, that looks broken. Can someone fix?

On Thu, Apr 9, 2015 at 9:50 AM, Joseph Silber <notifications@github.com>
wrote:

> In src/Illuminate/Database/Schema/Builder.php
> <https://github.com/laravel/framework/pull/8346#discussion_r28068286>:
>
> > @@ -68,6 +68,27 @@ public function hasColumn($table, $column)
> >  	}
> >
> >  	/**
> > +	 * Determine if the given table has given columns.
> > +	 *
> > +	 * @param  string  $table
> > +	 * @param  array   $columns
> > +	 * @return bool
> > +	 */
> > +	public function hasColumns($table, array $columns)
> > +	{
> > +		$tableColumns = $this->getColumnListing($table);
> > +
> > +		array_map('strtolower', $tableColumns);
>
> Doesn't array_map return a new array?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/framework/pull/8346/files#r28068286>.
>